lib/moonshot/dynamic_template.rb

```diff
@@ -24,12 +24,15 @@ def expose_binding
 
     def initialize(source:, parameters:, destination:)
       @source = File.read(source)
-      @parameters = Parameters.new(parameters)
+      @parameters = parameters
       @destination = destination
     end
 
     def process
       validate_destination_exists
+      @parameters = Parameters.new(
```

Comment:
please change it in backwards compatible way (accepting both string and lambda)

```ruby

def parameters_file
  @parameters.respond_to?(:call) ? @parameters.call(Moonshot.config.environment_name) : @parameters
end

```

Reply:
Improvement done.
Now it accepts both: lambda and array of params.
Thanks.